packages/taquito-rpc/src/types.ts

```diff
@@ -1499,6 +1500,23 @@ export interface ConstantsResponseCommon {
 
 export type Ratio = { numerator: number; denominator: number };
 
+export interface TicketUpdates {
+  ticket_token: {
+    ticketer: string;
+    content_type: MichelsonV1Expression;
+    content: MichelsonV1Expression;
+  };
+  updates: {
+    account: string;
+    amount: BigNumber;
+  }
+}
+
+export interface ConstantsResponseProto015 extends ConstantsResponseProto014 {
+  minimal_stake: BigNumber;
+  ticket_updates: TicketUpdates[];
```

Comment:
`ticket_updates` property is not in the correct place.

Reply:
See https://gitlab.com/tezos/tezos/-/blob/master/docs/api/lima-openapi.json?plain=1
![image](https://user-images.githubusercontent.com/65877347/199847972-d016d3d0-2dd7-428b-a0a3-8be2c4365905.png)